### ISSUE TITLE
ODataUriExtensions.BuildUri ignores $compute

### ISSUE THREAD

**Issue by voronov-maxim** [OData/odata.net#1132]
ODataUriExtensions.BuildUri does not add $apply to the uri.

### Assemblies affected
Microsoft.OData.Core, Version=7.4.3.20321

### Reproduce steps
```c#
[Fact]
public void BuildUrlWithCompute()
{
    var query = new Uri("http://gobbledygook/People?$compute=LifeTime mul 2 as mulLifeTime");
    var parser = new ODataUriParser(HardCodedTestModel.TestModel, ServiceRoot, query);
    ODataUri uri = parser.ParseUri();

    Uri res = uri.BuildUri(ODataUrlKeyDelimiter.Parentheses);
    Assert.Equal(query, res);
}
```
### Expected result
uri should be http://gobbledygook/People?$compute=LifeTime mul 2 as mulLifeTime")

### Actual result
uri is http://gobbledygook/People

**Comment by AlanWong-MS** [comment]
Related: #1045

**Comment by voronov-maxim** [comment]
fixed in commit bf18e56807babf55e360233f526cfdf7a09d16e9
